### DISCUSSION TITLE
Can I use this repo to build firmware for Ender 3 V1 using the 4.2.7 board?

### DISCUSSION THREAD

**Discussion by rajhlinux** [mriscoc/Ender3V2S1#1023] [Q&A]
Hello,

Is it possible I can use the mriscoc's firmware for Ender 3 V1 which has BL-Touch V3 and 4.2.7 motherboard?

I have a generic laser module and would like to use it so that I can make PCBs.

I know that the Ender V2 has a touch LCD display but the Ender V1 has a 128x64 LCD display.

Thanks.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/wiki#requirements
